plugins/org.eclipse.papyrus.uml.interaction.model/src/org/eclipse/papyrus/uml/interaction/model/spi/DiagramHelper.java

```diff
@@ -125,4 +125,35 @@ Command createMessageConnector(Supplier<Message> message, //
 	 */
 	Command deleteView(EObject diagramView);
 
+	public interface ViewTypes {
```

Comment:
I'm not sure if this is the optimal location for the centralization component ( I followed the apporach which was used in #308 for the modifier literals)

Reply:
I would keep it in as a separate interface, but only for historical reasons. I do not see any pros or cons for this being a separate interface.
I do not see however the removal of RepresentationKind class in the commit, with regards to the description of PR. Did I miss something?

Reply:
You are right seems like I forgot to stage the RepresentationKind remove change. Thanks for catching that.